Sources/WalletConnectModal/Modal/Screens/WalletList.swift

```diff
@@ -188,50 +185,15 @@ struct WalletList: View {
                 $0.onTapGesture {
                     withAnimation {
                         navigateTo(.walletDetail(wallet))
+                        
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```

Comment:
why we need delay?

Reply:
for slightly better UX, users will see `Continue in WalletXYZ` briefly and then it actually performs the deeplink

Reply:
left comment explaining this